samples/matter/lock/src/app_task.cpp

```diff
@@ -112,8 +134,10 @@ CHIP_ERROR AppTask::Init()
 	sStatusLED.Init(DK_LED1);
 	sLockLED.Init(DK_LED2);
 	sLockLED.Set(BoltLockMgr().IsLocked());
+#if !CONFIG_BOARD_NRF7002DK_NRF5340_CPUAPP
 	sUnusedLED.Init(DK_LED3);
```

Comment:
nit: Not related.. but why do we even bother about LED3 and LED4 if they are not used?

Reply:
Alternatively if those LEDs are used (e.g. to indicate factory reset) why they are called unused :)

Reply:
That's fair point, naming might not be the best in this case. Should I address it in this PR or maybe plan some sample cleanup task to not change only one sample and lead to further discrepancies between the samples?

Reply:
Its minor, both works. Just commented on this as you rewrote some led/button code.

Reply:
I think Marcin did a helper in window covering example for grouping LEDs, so at some point we should use it in all samples and only change the group initialization code for different boards. IMHO the goal should be to have as uniform code for all platforms as possible (we tend to praise "one code, multiple platforms" feature of the NCS, so we should set a good example ;)).

Reply:
It's here:
https://github.com/project-chip/connectedhomeip/blob/master/examples/platform/nrfconnect/util/include/LEDUtil.h#L24